nservicebus/rabbitmq/cluster.md

```diff
@@ -3,13 +3,13 @@ title: Using a RabbitMQ cluster
 summary: Guidelines for using RabbitMQ in a cluster
 ---
 
-The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster in the client applications and instead use an approach such DNS or load balancing. The same is recommended when using the NServiceBus RabbitMQ transport. For more details see the guide.
+The [RabbitMQ Clustering Guide](https://www.rabbitmq.com/clustering.html) recommends that client applications should not bake in the hostnames or IP addresses of the machines hosting nodes in a RabbitMQ cluster. An approach such as DNS or load balancing should be used instead. This approach is recommended when using the NServiceBus RabbitMQ transport. For more details, see the guide.
```

Comment:
> "DNS or load balancing" is it possible to link somewhere for these topics?

Hmm, not sure on that. Their doc just mentions them and basically says "these things are out of scope for us to explain"

I guess I could try and find some generic docs on tcp load balancers or a dynamic dns setup?